README.md

```diff
@@ -62,16 +62,50 @@ The aim of this project is to create a personalized bot to help with server mana
 ## Discord commands
 
```

Comment:
W rubryce `Discord Commands` wypisałbym raczej komendy discordowe, czyli bezpośrednio wpisywane do bota, czyli na przykład:

- `\help` - shows  bot usage instruction
- `\server` - Reads the message from the chat and returns embedded message with server stats

Reply:
Zrobione